specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2024-01-01-preview/appplatform.json

```diff
@@ -12780,6 +12793,11 @@
           "type": "string",
           "readOnly": true
         },
+        "endTime": {
+          "description": "End time of the deployment instance",
```

Comment:
nit: just curious will this property be valid for both long running and short lived app?

Reply:
Yes, it's applicable for both.